Date.cpp

```diff
@@ -4,7 +4,7 @@
 
 
 
-date::date(std::string init)
+date::date(std::string init, bool AUC)
```

Comment:
no, no, don't invalidate existing code. Rather make a second constructor that takes one bool extra, and the first one internally sets AUC to false by default.

Reply:
Overloading will work too, but it's not invalidating. This is compatible with current usages, as it's a default param.

Reply:
I see that, and it's true, but when you get code autocompletion in VS, having 2 constructors displays nicely and you can see your options. I'd still prefer 2.

Reply:
Review comment seconded.